Gemfile.lock

```diff
@@ -203,9 +198,6 @@ GEM
       multipart-post (>= 1.2, < 3)
     fastimage (2.1.0)
     ffi (1.9.18)
```

Comment:
Not a complete review, I just found that ffi and scrypt are only dependencies of authlogic and should also be removed from `Gemfile.lock`. A `bundle install` should do it.

Reply:
Once we removed ffi and pulled from master, we can stop ignoring the vulnerability CVE-2018-1000201 (#3425).